__tests__/app.test.js

```diff
@@ -33,4 +33,27 @@ beforeEach(() => {
           expect(res.body).toEqual({"msg": "Invalid Path"})
           });
     })
-  });
\ No newline at end of file
+  });
+
+describe('GET /api/reviews/:review_id', () => {
+    it('responds with the specified reivew and assiocated data', () => {
+        return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then((res) => {
+            expect(Object.keys(res.body.review)).toEqual(['review_id', 'title','category','designer','owner','review_body','review_img_url','created_at','votes'])
+            expect(res.body.review["review_id"]).toBe(1)
+          });
+    })
+  });
+
+describe('GET /api/review/:review_id', () => {
+  it('responds with an error when the review is not in the database', () => {
+      return request(app)
+      .get('/api/reviews/1234')
+      .expect(404)
+      .then((res) => {
+          expect(res.body).toEqual({"msg":"No review found for review_id: 1234"})
+      })
+  })
```

Comment:
This error handling looks spot on. Take a look at the case when you're id isn't a number but looking good otherwise.